controller/src/test/resources/org/jboss/as/controller/persistence/yaml/simple_list_add.yml

```diff
@@ -0,0 +1,10 @@
+wildfly-configuration:
+    extension:
+      org.jboss.as.failure:
+        module: org.jboss.as.failure
+    list:
+      my-list: 
+        strings: !list-add
```

Comment:
have you defined somewhere all the YAML tags that are supported or do users need to know about https://github.com/wildfly/wildfly-core/pull/4766/files#diff-d50b9b2a6b6cbf891246ead8a48357b7fdd24ef0ba76f35bda6e992afb26c742R535-R537?

Reply:
In the doc only